### PR TITLE
fix: fix deb10 publishing

### PR DESCRIPTION
**Pull request by gssbzn**
Fix deb publishing to publish the correct bin

### CODE REVIEW

File: tools/genevergreen/generate/publish.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ var distros = map[string]Platform{
 		architectures: []string{x86_64, aarch64},
 	},
 	"debian10": {
-		extension:     rpm,
```
Comment (gssbzn) [inline tools/genevergreen/generate/publish.go:56]: 🙃

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: tools/genevergreen/generate/publish.go (hunk 1 above)
Comment (fmenezes) [reply tools/genevergreen/generate/publish.go:56]: oh, was that always wrong?

Comment (gssbzn) [reply tools/genevergreen/generate/publish.go:56]: I think so

**Pull request merged**